Packs/ShiftManagement-AssignToNextShift/Playbooks/playbook-Assign_Active_Incidents_to_Next_Shift.yml

```
@@ -169,9 +169,13 @@ tasks:
     skipunavailable: false
     task:
       brand: ""
-      description: "Randomly assigns the incidents to users on call (requires shift\
-        \ management) and users on call.  \n\nhttps://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.10/Cortex-XSOAR-Administrator-Guide/Shift-Management#idf554fd0f-f93b-40cd-9111-1393bf25ac6e\n\
-        \nIncident Ids should be passed in as a comma separated list."
+      description: "Randomly assigns the incidents to users on call (requires shift management) and users on call.
+      
+        See for more information:  
+        [XSOAR 6.13](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.13/Cortex-XSOAR-Administrator-Guide/Shift-Management)  
+        [XSOAR 8](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Cloud-Documentation/Manage-roles-in-the-Cortex-XSOAR-tenant)
```

```suggestion
        [XSOAR 8 Cloud](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Cloud-Documentation/Manage-roles-in-the-Cortex-XSOAR-tenant)
```
What about on-prem?  https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/Manage-roles-in-the-Cortex-XSOAR-tenant